api/v3/Contact.php

```diff
@@ -56,7 +56,7 @@ function civicrm_api3_contact_create($params) {
   }
 
   if (!empty($params['dupe_check'])) {
-    $ids = CRM_Contact_BAO_Contact::getDuplicateContacts($params, $params['contact_type'], 'Unsupervised', array(), $params['check_permission']);
+    $ids = CRM_Contact_BAO_Contact::getDuplicateContacts($params, $params['contact_type'], 'Unsupervised', array(), $params['check_permissions']);
```

Comment:
Probably the line should be

   $ids = CRM_Contact_BAO_Contact::getDuplicateContacts($params, $params['contact_type'], 'Unsupervised', array(), !empty($params['check_permissions']));